src/Microsoft.ML.Data/Evaluators/Metrics/ClusteringMetrics.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Data
+{
+    /// <summary>
+    /// The metrics generated after evaluating the clustering predictions.
+    /// </summary>
+    public sealed class ClusteringMetrics
+    {
+        /// <summary>
+        /// Normalized Mutual Information
+        /// NMI is a measure of the mutual dependence of the variables.
+        /// <a href="http://en.wikipedia.org/wiki/Mutual_information#Normalized_variants">Normalized variants</a> work on data that already has cluster labels.
+        /// Its value ranged from 0 to 1, where higher numbers are better.
+        /// </summary>
+        public double Nmi { get; }
+
+        /// <summary>
+        /// Average Score. For the K-Means algorithm, the 'score' is the distance from the centroid to the example.
```

Comment:
>' [](start = 58, length = 1)

The apostrophes were already there, but we might need to XML encode them, here and below.